codesign/lib/src/transfer_utils.dart

```diff
@@ -0,0 +1,92 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:file/file.dart';
+import 'package:process/process.dart';
+
+import './utils.dart';
+
+/// Utility function class to handle upload/download of files from/to google cloud.
+class EngineArtifactTransfer {
```

Comment:
Based on the dart doc, a clearer name would be `GoogleCloudStorage`

Reply:
Sure, renamed.